ami/jobs/management/commands/process_single_image.py

```
@@ -0,0 +1,153 @@
+        # Submit the job
+        self.stdout.write("")
+        self.stdout.write(self.style.WARNING("Submitting job..."))
+
+        try:
+            image = SourceImage.objects.get(pk=image_id)
+            job = process_single_source_image(
+                source_image=image,
+                pipeline=pipeline,
+                run_async=not wait,
+            )
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)